checklists/apim_checklist.en.json

```diff
@@ -144,13 +164,53 @@
         {
             "category": "Management",
             "subcategory": "Business continuity and disaster recovery",
-            "text": "Deploy at least two scale units spread over two availability zones per region for best availability and performance",
+            "text": "Leverage multi-region deployment of API gateways (available only in premium tiers). APIM routes the requests to a regional gateway based on the lowest latency. You could use your own Traffic Manager with custom routing rules.",
+            "waf": "Reliability",
+            "service": "APIM",
+            "guid": "8d9b953c-cc0d-42c6-8e4f-24ef372e63c5",
+            "severity": "High",
+            "link": "https://learn.microsoft.com/azure/api-management/"
+        },
+        {
+            "category": "Management",
+            "subcategory": "Request Routing",
+            "text": "In multi-region model, use Policies to route the requests to regional backends based on availability or latency.",
+            "waf": "Reliability",
+            "service": "APIM",
+            "guid": "1b8d68a4-66cd-44d5-ba94-3ee94440e8d6",
+            "severity": "Medium",
+            "link": "https://learn.microsoft.com/azure/api-management/"
+        },
+        {
+            "category": "Management",
+            "subcategory": "Business continuity and disaster recovery",
+            "text": "Deploy at least two scale units per region for best availability and performance",
             "service": "APIM",
             "waf": "Reliability",
             "guid": "9c8d1664-dd9a-49d4-bd83-950af0af4044",
             "severity": "Medium",
             "link": "https://learn.microsoft.com/azure/api-management/high-availability"
         },
+        {
+            "category": "Management",
+            "subcategory": "Self-Hosted",
+            "text": "Its your responsibility to manage the scalability of self-hosted APIM gateways.",
+            "service": "APIM",
+            "waf": "Reliability",
+            "guid": "10f58602-f0f9-4d77-972a-956f6e0f2600",
+            "severity": "High",
+            "link": "https://learn.microsoft.com/en-us/azure/api-management/self-hosted-gateway-overview"
+        },
+        {
+            "category": "Management",
+            "subcategory": "Business continuity and disaster recovery",
+            "text": "Leverage Availability Zones where regionally applicable",
+            "waf": "Reliability",
+            "service": "APIM",
```

Comment:
This recommendation will be redundant to 9c8d1664-dd9a-49d4-bd83-950af0af4044 as there is no concept of just enabling AZs.

Reply:
Removed